sa/proto/sa.proto

```diff
@@ -343,6 +344,7 @@ message GetRevokedCertsRequest {
   int64 expiresAfterNS = 2; // Unix timestamp (nanoseconds), inclusive
   int64 expiresBeforeNS = 3; // Unix timestamp (nanoseconds), exclusive
   int64 revokedBeforeNS = 4; // Unix timestamp (nanoseconds)
+  int64 shardIdx = 5;
```

Comment:
I think this needs a comment explaining that it turns on querying from the new `revokedCertificates` table and it MUST NOT be turned on until that table is filled for all unexpired certificates (i.e. 90 days after that table has started filling).